File: Excel_Programs/Java/question8.java
import java.util.*;
class question8 
{
    public static void main(String[] args) 
    {
        Scanner sc = new Scanner(System.in);
        int n,sum=0;
        System.out.print("Enter the number of elements to be entered : ");
        n= sc.nextInt();
        int arr[] = new int[n];
        for(int i = 0; i<n;i++)
        {
            System.out.print("Enter a number : ");
            arr[i] = sc.nextInt();
        }
        for(int i=0;i<n;i++)
        {
            sum+=arr[i];
        }
        System.out.println(sum);

    }
}

File: Excel_Programs/C++/question8.cpp
#include<iostream>
using namespace std;

int main()
{
    int n;
    cout<<"Enter the number of elements to be entered : ";
    cin>>n;
    int arr[n];
    int sum=0;
    for(int i =0;i<n;i++)
    {
        cout<<"Enter a number : ";
        cin>>arr[i];
    }
    for(int i=0;i<n;i++)
    {
        sum +=arr[i];
    }
    cout<<sum;
    cout<<"\n";

}